server/testdb/seed.go

```diff
@@ -276,6 +278,49 @@ func SeedSource(db datastore.DatabaseClient, g *datastore.Group, uid string) (*d
 	return source, nil
 }
 
+func SeedSubscription(db datastore.DatabaseClient,
+	app *datastore.Application,
+	g *datastore.Group,
+	uid string,
+	groupType datastore.GroupType,
+	source *datastore.Source,
+	endpoint *datastore.Endpoint,
+	retryConfig *datastore.RetryConfiguration,
+	alertConfig *datastore.AlertConfiguration,
+	filterConfig *datastore.FilterConfiguration,
+) (*datastore.Subscription, error) {
+	if util.IsStringEmpty(uid) {
+		uid = uuid.New().String()
+	}
+
+	subscription := &datastore.Subscription{
+		UID:        uid,
+		GroupID:    g.UID,
+		Name:       "",
```

Comment:
Shouldn’t `name` also be an param of this func?

Reply:
Well the names are pretty useless in the context of the tests, so I didn't bother adding it

Reply:
Hm, okay cool